packages/storage/src/chrome/local.ts

```diff
@@ -1,25 +1,7 @@
 import { ExtensionStorage } from './base';
-import { KeyPrintJson } from '@penumbra-zone/crypto-web/src/encryption';
 import { UserChoice } from '@penumbra-zone/types/src/user-choice';
-import type { WalletJson } from '@penumbra-zone/types/src/wallet';
-
-export enum LocalStorageVersion {
-  V1 = 'V1',
-}
-
-export interface OriginRecord {
-  origin: string;
-  choice: UserChoice;
-  date: number;
-}
-
-export interface LocalStorageState {
-  wallets: WalletJson[];
-  grpcEndpoint: string;
-  passwordKeyPrint?: KeyPrintJson;
-  fullSyncHeight: number;
-  knownSites: OriginRecord[];
-}
```

Comment:
I moved the types that were defined `@penumbra-zone/storage/src/chrome/local` to `@penumbra-zone/types/src/local-storage`. 
This was to avoid a cyclic import which created a bug with the error `Cannot read properties of undefined (reading 'V1')` .   
This also created a lot of import changes in this PR.